package.json

```diff
@@ -10,28 +10,37 @@
     "storybook": "start-storybook -s src/app/assets -p 6006",
     "serve": "vite preview",
     "prepare": "husky install",
-    "test": "tsc && prettier --check . && eslint . && stylelint \"**/*.css\"",
+    "test": "tsc && prettier --check . && eslint . && stylelint \"**/*.css\" && npm run jest",
     "build": "npm run server:build && npm run client:build && npm run storybook:build",
-    "start": "node dist/server.js"
+    "start": "node dist/server.js",
+    "jest": "jest"
   },
   "devDependencies": {
     "@babel/core": "^7.14.3",
```

Comment:
Maybe you want to deinstall babel core? We used `npm remove @babel/core`

Reply:
Thanks, I did it 👍